miden-tx/src/kernel_tests/test_prologue.rs

```diff
@@ -273,8 +274,11 @@ fn consumed_notes_memory_assertions(
         let note = input_note.note();
 
         assert_eq!(
-            read_root_mem_value(process, CONSUMED_NOTE_SECTION_OFFSET + 1 + note_idx),
-            note.nullifier().as_elements(),
+            Nullifier::from(read_root_mem_value(
```

Comment:
comparing the nullifier instead of the felts because the former has hex-formatting